### PR TITLE
Disable red pin for Viewer Work Order Details page

### LINKED ISSUE #103: Remove red select location pin on Work Order Details Page - Viewable - view_2619
On this one, we shouldn't be displaying the Red select location pin, as we currently are. We should only see the fixed pins. Need to figure out a work around here that doesn't break the editor view

Screen Shot 2019-07-10 at 6 46 20 PM

### PR DESCRIPTION
**Pull request by mddilley**
Closes #103

### CODE REVIEW

File: knack/iframeMapMessenger.js [hunk 1]
```diff
@@ -34,7 +34,7 @@
     // Add React app as iframe if iframe doesn't already exist
     if ($(myView + " #mapIFrame").length === 0) {
       $(
-        '<iframe src="https://atd-geo-knack-ui.netlify.com/" frameborder="0" scrolling="yes" id="mapIFrame" \
+        '<iframe src="https://localhost:9001/" frameborder="0" scrolling="yes" id="mapIFrame" \
```
Comment (mateoclarke) [inline knack/iframeMapMessenger.js]: I don't think we want this referencing localhost in prod right @mddilley?

Comment (mateoclarke) [reply knack/iframeMapMessenger.js]: Right now in the knack js code (index.js), it references the "https://dnb4pix4gcpf6.cloudfront.net/atd-knack-signs-markings/98_marker_render/iframeMapMessenger.js". Do we need to update that?

File: src/SelectLocation.js [hunk 2]
```diff
@@ -368,6 +369,10 @@ export default class SelectLocation extends Component {
 
       switch (data.message) {
         case "SIGNS_API_REQUEST":
+          // Use workOrderDetailsViewable state to disable red pin and pulse if user is Viewer role
+          if (data.view === "view_2619") {
```
Comment (mateoclarke) [inline src/SelectLocation.js]: Nice, this is pretty clever. Kinda of a hard-coded hack but let's count our wins and never talk about this again.

File: knack/iframeMapMessenger.js (hunk 1 above)
Comment (mddilley) [reply knack/iframeMapMessenger.js]: Good catch! Just changed and pushed. I'll update the Knack JS in builder with the latest from master after merging this branch.

**Pull request merged**